actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -11,28 +11,61 @@ def self.remove_renderer(key)
     Renderers.remove(key)
   end
 
+  # RFC: Where is this used?
   class MissingRenderer < LoadError
```

Comment:
Actually it is on responders https://github.com/plataformatec/responders/blob/f0e093d01e799d8b67952c013daf8808e9ea02db/lib/action_controller/responder.rb#L209.